str/fine-mapping/finemap_runner.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+"""
+This script runs FINEMAP.
+
+analysis-runner --dataset "bioheart" --access-level "test" --description "Run finemap" --output-dir "str/associatr/fine_mapping" \
+     --image "australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:d4922e3062565ff160ac2ed62dcdf2fba576b75a-hail-8f6797b033d2e102575c40166cf0c977e91f834e" \
```

Comment:
afaik this is the default driver image, so you'll get this without specifying an image. Unless this points to a specific non-current image?